### PR TITLE
fix: updated zombienet to work with polkadot-sdk nodes

### PR DESCRIPTION
**Pull request by bee344**
Currently, there is an issue with pre-open hrmp channels when using nodes from polkadot-sdk ([PR](https://github.com/paritytech/polkadot-sdk/pull/1616)), which caused Trappist and KAH not producing blocks when using `medium-network.toml`. 

A workaround that is to remove `[[hrmp_channels]]` from the zombienet config and using the [script itself to open the channels](https://github.com/paritytech/polkadot-sdk/pull/1352/commits/a3d1264b86368788bdf4ffc1217fc4b3787dac99#diff-4184a0536459d629f83caaeb522651fd0e79cf61f33dd4d5660c92b363442050R203-R230).

Edit: typo

### CODE REVIEW

File: scripts/testNetworkLiquidAssets.ts [hunk 1]
```diff
@@ -87,15 +109,15 @@ const main = async () => {
...
+	await api.tx.utility.batch(txs).signAndSend(alice, { nonce: -1 });
 
 	await delay(24000);
 
 	const nextLpToken = await api.query.assetConversion.nextPoolAssetId();
 
 	const lpToken = Number(nextLpToken) - 1;
 
-	logWithDate(chalk.yellow('Asset and Liquidity Pool created'));
+	logWithDate(chalk.yellow('Asset and Liquidit Pool created'));
```
Comment (marshacb) [inline scripts/testNetworkLiquidAssets.ts:120]:
```suggestion
	logWithDate(chalk.yellow('Asset and Liquidity Pool created'));
```

File: scripts/util.ts [hunk 2]
```diff
@@ -57,3 +57,25 @@ export const awaitBlockProduction = async (wsUrl: string) => {
 		logWithDate(chalk.blue('Polkadot-js successfully disconnected'));
 	});
 };
+
+export const awaitEpochChange = async (api: ApiPromise) => {
```
Comment (marshacb) [inline scripts/util.ts:61]: I see the ending logs reference HRMP channels being opened. Will we likely exclusively use this for waiting to open HRMP channels?

**Review by marshacb — APPROVED** [review]
small nits. LGTM!

File: scripts/util.ts (hunk 2 above)
Comment (marshacb) [reply scripts/util.ts:61]: Cool, I understand that part of it. My question was more about semantics given the functions name and the mentioning of HRMP channels in its logs. Not a big deal right now and can always be revisited :)

**Review by TarikGul — APPROVED** [review]
LGTM, great job!

**Pull request merged**